### ISSUE TITLE
Configurable menu names 

### ISSUE THREAD

**Issue by cmsu224** [cmsu224/clan-events#19]
A couple of other clans who use this plugin want to remove the 'past' from 'past skill of the week' as they don't track history.  And also change some other names like 'clan events' to 'clan hub'.  Maybe there is a way to set the name from the Google Sheet

**Comment by Brianmm94** [comment]
Yeah, I was thinking about some of this after the changes were made. I wasn't sure how other clans would be using it. The options are to switch the names to be more general or to see if there's a way to pull the names from a spreadsheet in addition to the data in a single http request.
